department-of-veterans-affairs/va.gov-cms
# VACMS-10466 increase jsonapi response max size for next

## Description

Relates to #10466. 
Companion PR to https://github.com/department-of-veterans-affairs/content-build/pull/1295

## Testing done
in next-build, see that `yarn export` returns ~1600 nodes instead of ~950 without this override.

## Screenshots
n/a, sorry

## QA steps
in the next-build repo, point .env at your local ddev instance like so:
```
in .env.local: 
NEXT_PUBLIC_DRUPAL_BASE_URL=http://va-gov-cms.ddev.site
```

run `yarn export`. see that ~950 nodes are built.

now apply this service override in your local cms repo, clear cache

run `yarn export` in the next-build repo again, see that 1600 nodes are built. hooray!
